src/main/java/com/revature/repository/CommentRepository.java

```diff
@@ -19,6 +20,7 @@
     /**
      * @see returns all comments based on interviewId
      */
-	List<Comment> findAllByInterviewId(Interview interviewId);
-
+//	List<Comment> findAllByInterviewId(Interview interviewId);
+	
+	List<Comment> findAllByInterviewInterviewer (Interview interviewId, InterviewInterviewer interviewInterviewer);
```

Comment:
Has this been tested? From my experience with Spring Data, it seems like this combination of method name and parameters is too ambiguous for Spring Data.